FreeRTOS-Plus/Source/FreeRTOS-Plus-TCP/History.txt

```diff
@@ -1,3 +1,47 @@
+Changes between V2.2.1 and V2.2.2
+	+ Add CBMC proofs for 4 functions.
+	+ Address ~1200 MISRA violations.
+	+ Support caching multiple IP addresses per DNS entry in TCP library.
+	+ Security improvements in the TCP library.
+	+ Add additional length check when checksum offloading is used.
+	+ Add ipconfigIP_PASS_PACKETS_WITH_IP_OPTIONS compile time constant to enable
+	  users to drop packets with IP options (IP Options are not supported).
+	+ Introduce ipconfigUDP_PASS_ZERO_CHECKSUM_PACKETS macro which enables the
```

Comment:
Say why, as per my previous comment "improve security by allows the user to drop UDP packets that have a checksum of zero even though the UDP specification says zero checksus should be accepted.".